src/pip/_internal/cache.py

```diff
@@ -117,7 +117,7 @@ class WheelCache(Cache):
     def __init__(self, cache_dir, format_control):
         super(WheelCache, self).__init__(cache_dir, format_control, {"binary"})
 
-    def get_path_for_link(self, link):
+    def get_path_for_link(self, link, ephem=False):
```

Comment:
Instead of having a boolean in the signature, I'd suggest you implement a new method `WheelCache.get_ephem_path_for_link`.

Relevant Blog Post: https://martinfowler.com/bliki/FlagArgument.html